rust/worker/src/segment/metadata_segment.rs

```diff
@@ -177,7 +182,7 @@ impl<'me> MetadataSegmentWriter<'me> {
         let full_text_index_reader = match (pls_reader, freqs_reader) {
             (Some(pls_reader), Some(freqs_reader)) => {
                 let tokenizer = Box::new(TantivyChromaTokenizer::new(Box::new(
-                    NgramTokenizer::new(1, 3, false).unwrap(),
+                    NgramTokenizer::new(3, 3, false).unwrap(),
```

Comment:
OOC - why was 1 wrong? I think trigram makes sense since it matches singlenode

Reply:
discussed offline. This generates token of length 1, 2 and 3 but instead the search algorithm relies only on token of length 3.